packages/ui/package.json

```diff
@@ -16,6 +16,7 @@
     "react": "17.0.2"
   },
   "dependencies": {
+    "@chakra-ui/icons": "1.0.14",
```

Comment:
I needed sth for "hamburgerMenu" icon and if we use chakra anyway, so it won't be a problem I thought

Reply:
We use our own set of icons instead of Chakra ones. My bad because I didn't add the burger icon to Figma. Now you can find it [here](https://www.figma.com/file/ur0KLA9ZOtiodrAmpXrxh9/CodersCamp-app-and-website?node-id=1082%3A3353). Copy both solid and outlined versions. As they are the same you can use any of them 😄